content/docs/programs/requisitions.md

```diff
@@ -61,7 +61,66 @@ There are several areas where you'll see additional information when using a pro
+enable these extra columns as described in the [mSupply Desktop
+documentation](https://docs.msupply.org.nz/other_stuff:virtual_stores#preferences_tab).
+
+![Show extra fields on requisitions preference](/docs/programs/images/show_extra_fields_on_requisitions.png)
+
+This is how the extra columns look within an Internal Order:
+
+![](/docs/programs/images/internal_order_extra_fields.gif)
+
+The line edit page will also contain the extra columns as well:
+
+<!-- TODO: Calculations (currently aren't calculated)  -->
+
+| Column                    | Description                                                                                                  |
+| :------------------------ | :----------------------------------------------------------------------------------------------------------- |
+| **Initial stock on hand** | The customer's stock on hand at the start of the program period                                              |
+| **Incoming**              | The customer's incoming stock in units for this item                                                         |
+| **Outgoing**              | The customer's outgoing stock in units for this item                                                         |
+| **Losses**                | Inventory Adjustment                                                                                         |
+| **Additions**             | Inventory Adjustment                                                                                         |
+| **Available**             | Initial stock on hand + incoming stock +/- inventory adjustment - outgoing stock                             |
+| **Short Expiry**          | Sum of available stock for the item which has an expiry date after the program end date                      |
+| **Days out of stock**     | (Target MOS \* AMC) - available stock                                                                        |
+| **AMC**                   | The customer's average monthly consumption for this item                                                     |
+| **MOS**                   | Months of stock for the item                                                                                 |
+| **Reason**                | A reason will be provided if their is a difference between the suggested quantity and the requested quantity |
+
+![Program Requisition Line
+Edit](/docs/programs/images/internal_order_extra_fields_edit.png)
+
+If your store has the store preference `Use consumption &
+stock from customers for internal orders` enabled
+
+![Use consumption & stock from customers for internal
+orders](/docs/programs/images/consumption_data_pref.png)
+
+Then you will see an extra table underneath the input fields which shows the
+customer's data based on their program requisitions for that period.
+
+![](/docs/programs/images/internal_order_item_info.png)
+
+The table has 6 columns:
```

Comment:
tbh took me a second to realise this was related to the table in the screenshot above... maybe:

```suggestion
This customer data table has 6 columns:
```